docs/tutorials/jokes.md

```diff
@@ -88,23 +88,12 @@ If you're planning on using CodeSandbox, you can use [the Basic example][the-bas
 npx create-remix@latest
 ```
 
-<docs-info>
```

Comment:
I think this info block needs to be kept, as the user will be prompted as described.

Reply:
I was checking it on Windows and it wasn't happening to me but changed it to Linux and looks like it will be prompted the first time a dev tries to install/run the command `npx create-remix@latest` and the following installs doesn't have the prompt. Will update it now, thanks for raising it